ppb/events.py

```diff
@@ -145,6 +174,63 @@ class Render:
     scene: Scene = None
 
 
+@dataclass
+class ReplaceScene:
+    """
+    Fired to replace the current scene with a new one.
+
+    new_scene can be an instance or a class. If a class, must include kwargs.
+    If an instance kwargs must be falsey, default is None.
+
+    Examples:
+        * `signal(new_scene=ChangeScene(MyScene(player=player))`
+        * `signal(new_scene=ChanngeScene, kwargs={"player": player}`
+    """
+    new_scene: Union[Scene, Type[Scene]]
+    kwargs: Dict[str, Any] = None
+    scene: Scene = None
+
+
+@dataclass
+class SceneContinued:
+    """
+    Fired when a paused scene continues.
+    """
+    scene: Scene = None
+
+
+@dataclass
+class SceneStarted:
+    """
+    Fired when a scene starts.
+    """
+    scene: Scene = None
+
+
+@dataclass
+class SceneStopped:
+    """
+    Fired when a scene stops.
+    """
+    scene: Scene = None
+
+
+@dataclass
+class ScenePaused:
+    """
+    Fired when a scene pauses.
+    """
+    scene: Scene = None
+
+
+@dataclass
+class StopScene:
+    """
+    Fired to stop a scene.
+    """
```

Comment:
I think the above comments make it clear that more documentation around the scene life cycle events is needed.